packages/forklift-console-plugin/src/modules/Providers/ProvidersWrapper.tsx

```diff
@@ -3,6 +3,14 @@ import { withModalProvider } from '@kubev2v/common/polyfills/sdk-shim';
 
 import ProvidersPage from './ProvidersPage';
 
+if (process.env.DATA_SOURCE === 'mock') {
```

Comment:
I don't like this location for standing up the mock service worker.

I'm looking in to see how/if a different part of console can be hooked to make it more global/stable.  Not sure if it is even possible, but we need to know if we want to be able to mock out the base console as well (for a fully "static" console app)

Reply:
@sjd78  this is the entry point that we get from the console.  However even here it doesn't work - more research is needed.

Reply:
Yeah, I've been looking in to how best to do this bit.

Getting it to work on a per-page or per-exposed module may be tricky.